src/include/lean/lean.h

```diff
@@ -1927,6 +1927,70 @@ static inline size_t lean_data_hashmap_mk_idx(lean_object* sz, uint64_t hash) {
     return (size_t)(hash & (lean_unbox(sz) - 1));
 }
 
+static inline size_t lean_data_hashset_mk_idx(lean_object* sz, uint64_t hash) {
```

Comment:
```suggestion
static inline size_t lean_hashset_mk_idx(lean_object* sz, uint64_t hash) {
```

Similarly, I would just removing the `data` prefix from this name.